src/store/actionCreators/studyPage.tsx

```diff
@@ -0,0 +1,11 @@
+import { ThunkAction } from 'redux-thunk';
+import { RootState } from '../rootState';
+import { Sqon } from '../sqon';
+import { setActiveSqonIndex, setSqons } from './virtualStudies';
+
+export const createQueryInCohortBuilder = (
+  sqons: Sqon[],
+): ThunkAction<void, RootState, null, any> => async (dispatch) => {
```

Comment:
ThunkAction<void, RootState, null, any> instead of any you could be more specific by saying what actions are used.